catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -836,4 +846,28 @@ private static void assertDateAttribute(
       assertDateAttribute(reason, attribute, expectedDateTimes);
     }
   }
+
+  public static class NitfValue {
```

Comment:
:pencil2: If this isn't be designed for extension, we should mark it as `final`

Reply:
Fixed.